www/js/splash/startprefs.js

```diff
@@ -230,8 +230,8 @@ angular.module('emission.splash.startprefs', ['emission.plugin.logger',
       return startprefs.getPendingOnboardingState().then(function(result){
         if (result == null) {
           var temp = ReferralHandler.getReferralNavigation();
-          if (temp == 'goals') {
```

Comment:
I would suggest reverting the changes to this file.
I have a large pending restructure of the file (https://github.com/e-mission/e-mission-phone/pull/470) and you are going to get a merge conflict.

Reply:
And this code is to deal with referrals, which is not really an issue for you right now.